src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -144,4 +145,53 @@ public void testNegativeLoginValidationInvalidEmail(){
                                     .getError()).as("Email id Validated")
                                     .isEqualTo("The email is in an invalid format.");
     }
+
+    @Test
+    public void testNegativeLoginAuthenticationWrongAppKey(){
+        com.training.pojos.ga.authentication.Request request =
+                new com.training.pojos.ga.authentication.Request(email,password);
+        String errorCode = "COMMONS-0001";
+        headerMap.put(map.get("gaValidationAppKeyHeaderName"),
+                RandomStringUtils.randomAlphanumeric(map.get("gaValidationAppKeyHeaderValue").length()));
```

Comment:
Lets rename this appKey in json to - `gaAppKeyHeaderValue` since its been used for all ga tests. Also align all your keys in testData.json as per this